.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Title
+[Provide a succinct and descriptive title for the pull request]
+
+## Type of Change
+- [ ] Bug fix
+- [ ] New feature
+- [ ] Documentation update
+- [ ] Refactoring
+- [ ] Hotfix
+- [ ] Security patch
```

Comment:
![Screenshot from 2024-12-15 06-16-52](https://github.com/user-attachments/assets/9406ae10-dc3a-4607-8854-dac53d1d5d47)

It doesn't behave like a radio button. It acts like there are some remaining tasks to be done for the issue